### PR TITLE
Create the mentor card component

### LINKED ISSUE #6: Create the mentor card component
### Description
As part of this issue, we need to create the mentor card component, which should show the mentor's profile picture, name and headling. match the provided mockup. 
### Tasks

1. Set up the React components as required.
2. Implement the user story.
3. Integrate the necessary components and assets to achieve the desired functionality and visual appearance.
4. Ensure responsiveness and compatibility across different devices and screen sizes.

Mockup: https://www.figma.com/file/nehm2ff8aR7NkPerg39fK9/ScholarX-V2-%7C-Wireframes?type=design&node-id=523%3A6&mode=design&t=MusZHQ3y31nKxoli-1

Image

Note: You may want to mock the API refer to the [API doc](https://documenter.getpostman.com/view/27421496/2s93m1a4ac#47236625-31a9-4991-a9c4-cbee65ebf1d7) to see the relevant details. 

### Acceptance Criteria

- The component closely resembles the provided mockup.
- All required sections, elements, and content are present and displayed correctly.
- The component is responsive and works well on various devices and screen sizes.
- The implementation follows best practices and coding standards.
- Unit tests are written

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #6 

## Goals

- To create the mentor card 

## Approach

- Create mentor card component
- Create the story
- Configure the interactions addon
- Add the tests
- Update the github workflows

## Screenshots

Screenshot 2023-07-15 at 8 36 40 PM

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/components/MentorCard/MentorCard.component.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        className={styles.avatarContainer}
+      />
+    )}
+    <div className={styles.avatarContainer}>
+      <Title level={5} className={styles.title}>
+        {mentor.profile.first_name} {mentor.profile.last_name}
+      </Title>
+      <Text>{mentor.application.designation}</Text>
+      <br />
+      <Text type="secondary">Google Inc</Text>
```
Comment (Piumal1999) [inline src/components/MentorCard/MentorCard.component.tsx]:
```suggestion
      <Text type="secondary">{mentor.application.company_or_institution}</Text>
```

File: src/components/MentorList/MentorList.stories.ts [hunk 2]
```diff
@@ -1,44 +1,24 @@
...
 
 export default meta;
+
 type Story = StoryObj<typeof meta>;
 
 export const ListWithData: Story = {
   args: {
-    data: data
-  }
+    mentors: [...mentors, ...mentors],
```
Comment (Piumal1999) [inline src/components/MentorList/MentorList.stories.ts:21]: This is ok for now. But I think there might be a way to generate dummy data when using storybook. Something similar to [FakerJs](https://fakerjs.dev/)?

**Review by Piumal1999 — CHANGES_REQUESTED** [review]
Good job @anjula-sack! I requested a minor change.

File: src/components/MentorList/MentorList.stories.ts (hunk 2 above)
Comment (anjula-sack) [reply src/components/MentorList/MentorList.stories.ts:21]: This generates only specific fields if we can generate whole objects that's better.

**Review by Piumal1999 — APPROVED** [review]

**Pull request merged**